app/services/analytics_events.rb

```diff
@@ -84,6 +84,13 @@ def account_visit
     track_event('Account Page Visited')
   end
 
+  # @identity.idp.event_name Add Email: Email Confirmation
+  # @param [Hash] result results from an email confirmation
+  # A user has been sent an email confirmation
+  def add_email_confirmation(result)
```

Comment:
the goal is to try to document what hash fields. Looks like this comes from EmailConfirmationTokenValidator

Looks like these are some of the attributes: https://github.com/18F/identity-idp/blob/main/app/services/email_confirmation_token_validator.rb#L48-L52

Reply:
Got it - check now?